app/queues/certificate/challenge-completer-worker.server.ts

```diff
@@ -13,7 +19,84 @@ export const challengeCompleterWorker = new Worker<ChallengeCompleterData>(
   challengeCompleterQueueName,
   async (job) => {
     const { rootDomain } = job.data;
-    logger.info(`TODO complete challenges for ${rootDomain}`);
+    const childrenValues = await getChildrenValuesOfQueueName<OrderCreatorOutputData>({
+      queueName: orderCreatorQueueName,
+      job,
+    });
+
+    // Get the order creator return value
+    const [orderCreatorRetval] = Object.values(childrenValues);
+
+    logger.info('Attempting to complete ACME challenges with the provider', {
+      rootDomain,
+      certificateId: orderCreatorRetval.certificateId,
+    });
+
+    let letsEncrypt;
```

Comment:
Can you move this lower down, where you actually use it near 60?

Reply:
Sure!